src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -101,6 +109,12 @@ const SkillsQuizStepper = () => {
                 <Configure hitsPerPage={NUM_RESULTS_PER_PAGE} />
                 {skillQuizFacets}
               </InstantSearch>
+              <TagCloud
+                tags={selectedSkills}
+                onRemove={
+                  (skillMetadata) => dispatch(removeFromRefinementArray('skill_names', skillMetadata.title))
```

Comment:
Here we need to check that if there is only one value in array then we need to call `deleteRefinementAction` this function as well to remove this attribute from URL and Reducer. Otherwise, a value with 0 is appended in the dropdown.